target/linux/layerscape/armv8_64b/config-5.15

```diff
@@ -764,6 +765,10 @@ CONFIG_TREE_RCU=y
 CONFIG_TREE_SRCU=y
 # CONFIG_UACCE is not set
 CONFIG_UBIFS_FS=y
+CONFIG_UBIFS_FS_LZO=y
+CONFIG_UBIFS_FS_XATTR=y
+CONFIG_UBIFS_FS_ZLIB=y
+# CONFIG_UBIFS_FS_ZSTD is not set
```

Comment:
These options are already set in the generic configuration, no need to specify them here again.

Reply:
I removed these changes myself.